graphql.md

```diff
@@ -0,0 +1,1156 @@
+---
+title: 'GraphQL API'
+intro: 'The GraphQL API is a **read-only** API for delivering content from Statamic to your frontend, external apps, SPAs, and numerous other possible sources. Content is delivered as JSON data.'
+stage: 1
+pro: true
+id: fc564ddf-80c1-4d87-8675-4a41f13c7774
+---
+
+## Enable GraphQL
+
+Enable the GraphQL API in your config or with an environment variable.
```

Comment:
I've just tested this out but ran into an CSRF exception as I didn't add it to my `VerifyCsrfToken` middleware, maybe this should be documented here to ensure no one runs into it?

Reply:
You shouldn't have to do that. Where'd you get the error?

Reply:
When I visited the GraphQL Explorer in the Control Panel.

Reply:
Sounds like you don't have the rebing/graphql-laravel package installed.
I know you do a lot of PRs so this is probably because you just switched branches and didn't actually update composer dependencies.